kinetic/distribution.yaml

```
@@ -2414,11 +2414,20 @@ repositories:
       version: master
     status: maintained
   earth_rover_localization:
+    doc:
+      type: git
+      url: https://github.com/earthrover/earth_rover_localization.git
+      version: 1.0.0-1
```

This supports tags but we strongly recommend using a branch here so that you don't have to make a release to update the docs. 
```suggestion
      version: master
```

Note that this is not a valid tag either the only tags is [`1.0.0`](https://github.com/earthrover/earth_rover_localization/tree/1.0.0)